ui/src/pages/backoffice/EItem/EItemSearch/EitemList.js

```diff
@@ -0,0 +1,125 @@
+import { DocumentAuthors } from '@components';
+import { SearchEmptyResults } from '@components/SearchControls';
+import { DocumentIcon, EItemIcon } from '@pages/backoffice';
+import { BackOfficeRoutes } from '@routes/urls';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Grid, Header, Icon, Item, Label, List } from 'semantic-ui-react';
+import isEmpty from 'lodash/isEmpty';
+
+class EItemListEntry extends Component {
+  render() {
+    const { eitem } = this.props;
+
+    return (
+      <Item>
+        <Item.Content>
+          <Item.Header
+            as={Link}
+            to={BackOfficeRoutes.eitemDetailsFor(eitem.metadata.pid)}
+            data-test={`navigate-${eitem.metadata.pid}`}
+          >
+            <EItemIcon />
+            {eitem.metadata.document.title}{' '}
+          </Item.Header>
+          <Grid columns={4}>
+            <Grid.Column computer={6} largeScreen={5}>
+              <Item.Meta className={'document-authors'}>
+                <DocumentAuthors
+                  metadata={eitem.metadata.document}
+                  prefix={'by '}
+                />
+              </Item.Meta>
+              {eitem.metadata.doi && (
+                <>
+                  <label>DOI</label> {eitem.metadata.doi}
+                </>
+              )}
+            </Grid.Column>
+            <Grid.Column computer={4} largeScreen={4}>
+              <Item.Meta>
+                <List>
+                  <List.Item>
+                    {!isEmpty(eitem.metadata.files) && (
+                      <List.Content>
+                        {eitem.metadata.files.length} file(s) attached{' '}
+                        <Icon name="file" />
+                      </List.Content>
+                    )}
+                  </List.Item>
+                  <List.Item>
+                    <List.Content>
+                      {!isEmpty(eitem.metadata.urls) && (
+                        <List.Content>
+                          {eitem.metadata.urls.length} external providers <Icon name="external" />
+                        </List.Content>
+                      )}
+                    </List.Content>
+                  </List.Item>
+                </List>
+              </Item.Meta>
+            </Grid.Column>
+            <Grid.Column computer={4} largeScreen={4}>
+              <List>
+                <List.Item>
+                  <List.Content>
+                    {eitem.metadata.open_access && (
```

Comment:
Shall we display a `closed access` label in case it is not open?

Reply:
the question is which one will be less likely to happen, so the label does not appear everywhere, I assumed the open will be less frequent, do you think it might be an opposite case?